simple_email_confirmation/admin.py

```diff
@@ -1,10 +1,11 @@
 from django.contrib import admin
 
-from .models import EmailAddress
+from .models import get_email_address_model
```

Comment:
We should update this since the function moved to `__init__.py`

```suggestion
from simple_email_confirmation import get_email_address_model
```